galley/pkg/crd/validation/config.go

```diff
@@ -187,6 +187,10 @@ func rebuildWebhookConfigHelper(
 		if webhookConfig.Webhooks[i].NamespaceSelector == nil {
 			webhookConfig.Webhooks[i].NamespaceSelector = &metav1.LabelSelector{}
 		}
+		if webhookConfig.Webhooks[i].SideEffects == nil {
```

Comment:
This is only on the rebuild, do we need it in the original template? https://github.com/istio/istio/blob/9c1d128ed95e880922a82756f5555d12b2f72ae7/install/kubernetes/helm/istio/charts/galley/templates/validatingwebhookconfiguration.yaml.tpl#L67 It looks like we already have side effects set to none here?

Reply:
sure